hadoop-tools/hadoop-azure/src/main/java/org/apache/hadoop/fs/azurebfs/services/AbfsInputStream.java

```diff
@@ -583,8 +583,8 @@ public synchronized long skip(long n) throws IOException {
       newPos = 0;
       n = newPos - currentPos;
     }
-    if (newPos > contentLength) {
-      newPos = contentLength;
+    if (newPos >= contentLength) {
```

Comment:
what does skip(0) on a 0-byte file do? I don't think we've ever looked at that?

Reply:
skip(0) should seek to 0, corrected and added test